editor.Rmd

```diff
@@ -6,59 +6,66 @@
 This chapter outlines the workflow for an editor to review, publish and promote a blog post or tech note. It may also provide insights for post authors.
 ```
 
-## Review a Post {#review}
+## Review a post {#review}
 This section explains how to review a blog post or tech note.
 The editor for a post is typically the Community Manager or their intern.
 
-A post is submitted as a pull request on GitHub and we use the [well documented GitHub web interface](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request) to suggest edits, make comments, and ultimately merge the pull request to publish the post.
+A post is submitted as a pull request on GitHub. 
+We use the [well documented GitHub web interface](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests) to suggest edits, make comments, and ultimately merge the pull request to publish the post.
```

Comment:
Here it's fine to use well documented since it's a guide for the team (so folks who should be at ease with GitHub?).